inference-engine/cmake/config.cmake.in

```diff
@@ -4,6 +4,10 @@
 
 if(DEFINED IE_MAIN_SOURCE_DIR AND TARGET inference_engine)
     set(InferenceEngine_LIBRARIES inference_engine inference_engine_c_api)
+    add_library(IE::inference_engine ALIAS inference_engine)
+    if(TARGET inference_engine_c_api)
+        add_library(IE::inference_engine_c_api ALIAS inference_engine_c_api)
+    endif()
```

Comment:
should not we set like this?
```
set(InferenceEngine_LIBRARIES IE::inference_engine IE::inference_engine_c_api)
```

Reply:
In this case, I think, it doesn't matter. Just some 3rd-party projects use `InferenceEngine_LIBRARIES` variable, while others use `IE::inference_engine` directly.